ci/Makefile

```diff
@@ -21,8 +19,6 @@ push-hail-ci: build-hail-ci
 	docker push $(HAIL_CI_IMAGE)
 	echo $(HAIL_CI_IMAGE) > hail-ci-image
 
-restart-all-proxies: restart-proxy restart-batch-proxy
-
 restart-proxy: ZONE = $(shell gcloud compute instances list --filter=name=dk-test --format="value(ZONE)")
```

Comment:
What is this proxy for?

Reply:
GitHub makes http requests as a push notification of changes to PR state.